starcheck/utils.py

```diff
@@ -145,3 +154,41 @@ def get_run_start_time(run_start_time, backstop_start):
         else:
             raise ValueError("Float run_start_time should be negative")
     return ref_time.date
+
+
+@python_from_perl
+def config_logging(outdir, verbose, name):
+    """Set up file and console logger.
+    See http://docs.python.org/library/logging.html
+              #logging-to-multiple-destinations
+    """
+    # Disable auto-configuration of root logger by adding a null handler.
+    # This prevents other modules (e.g. Chandra.cmd_states) from generating
+    # a streamhandler by just calling logging.info(..).
+    class NullHandler(logging.Handler):
+        def emit(self, record):
+            pass
+    rootlogger = logging.getLogger()
+    rootlogger.addHandler(NullHandler())
+
+    loglevel = {0: logging.CRITICAL,
+                1: logging.INFO,
+                2: logging.DEBUG}.get(int(verbose), logging.INFO)
+
+    logger = logging.getLogger(name)
+    logger.setLevel(loglevel)
+
+    # Remove existing handlers if calc_ccd_temps is called multiple times
```

Comment:
This is no longer in calc_ccd_temps so I assume this comment should go.   Not 100% if the code is right for it as well but...